lib/search.js

```diff
@@ -96,12 +88,13 @@ module.exports = function (client, value, from, size, effect, weights) {
                     },
                     script_score: {
                         lang: 'groovy',
-                        script: 'pow(' +
+                        script: '(doc["name.raw"].value.equalsIgnoreCase(name)) ? 1 : _score * pow(' +
                         'doc["score.detail.popularity"].value * weightPopularity + ' +
                         'doc["score.detail.quality"].value * weightQuality + ' +
                         'doc["score.detail.maintenance"].value * weightMaintenance, ' +
                         'effect)',
                         params: {
+                            name: value,
```

Comment:
perhaps we can use `value.toLowerCase()` here and do a `equals()` instead. It will be slightly faster.